profiles/daily_trading/profile.json

```diff
@@ -36,7 +36,7 @@
     "profile": {
         "avatar": "default_profile.png",
         "description": "DailyTrading is a low risk versatile profile that catches buy and sell opportunities using RSI and moving average indicators.",
-        "id": "default",
+        "id": "daily_trading",
```

Comment:
So blank will be the default?

Reply:
yes, and onboarding will enable the user to change it

Reply:
:+1: